docker-bake.hcl

```diff
@@ -0,0 +1,186 @@
+variable "REGISTRY" {
+  default = "us-docker.pkg.dev"
+}
+
+variable "PROJECT_ID" {
```

Comment:
Does "PROJECT_ID" have to be a separate variable? It would reduce the differences vs. the upstream file if this was combined with one of the others, e.g. REPOSITORY="boba-392114/bobanetwork-tools-artifacts/images" or appending it to REGISTRY.

Reply:
good question.
I think the way it's structured now it provides common setting with cirleci (https://docs.docker.com/build/bake/reference/)

https://github.com/bobanetwork/v3-anchorage/pull/81/files/49e307515d82b92e88567a517425b485e4171aa9#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R236-R243

these exported variables get evaluated in the bake file.